packages/iobio-react-components/src/components/histogram.tsx

```diff
@@ -20,7 +20,7 @@
  */
 
 import { useEffect } from 'react';
-import { getBooleanAttributes, setElementStyles } from '../utils';
+import { getBooleanAttributes, setElementStyles } from '../utils/iobioTools.mts';
```

Comment:
Likewise I've used `tools` in a couple cases where I have a file containing several function definitions.

Basically a replacement for `utils`, but also we're outside the structure of a bigger app where we'd be describing a specific `router` or `controller` file.

Reply:
So I gave this some more thought and generally these are functions for interfacing with the Iobio Web Components.

Some are used in the React context and some are used in the Node context, but generally I think these functions can be moved to their respective places in the project instead of grouped here.

Reply:
I've moved several definitions into constants, several into statisticsTools, and renamed `iobioTools` as `webComponentTools` which is a more specific description.